var/spack/repos/builtin/packages/py-albumentations/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyAlbumentations(PythonPackage):
+    """albumentations is a fast image augmentation library and
+    easy to use wrapper around other libraries."""
+
+    homepage = "https://github.com/albu/albumentations"
+    pypi     = "albumentations/albumentations-1.1.0.tar.gz"
+
+    version('1.1.0', sha256='60b067b3093908bcc52adb2aa5d44f57ebdbb8ab57a47b0b42f3dc1d3b1ce824')
+    version('0.4.2', sha256='93baec3ca01a61bc81fa80563cdebf35dbae3f86b573e4cbe5c141c94782737f')
```

Comment:
Both of these versions are pretty old, is there a reason you don't only include 1.3.0?

Reply:
This was sitting in one of our local repos since 2019, updated this past summer as a dependency for `py-tiatoolbox` (PR coming at somepoint).